src/card/card.scss

```diff
@@ -0,0 +1,32 @@
+@import '../var-defaults.scss';
+@import '_attractions-theme.scss';
+
+.card {
+  box-shadow: $shadow-0;
+  background-color: #fff;
```

Comment:
And this, this also asks for a variable. Let it be `$card-bg`

Reply:
For a second thought, no, let's keep it white. I see no use in changing the card color for all cards, and individual cards can be styled on demand.

Reply:
I don't think people would commonly change the background color of individual cards, but would rather want to change all cards to conform to a certain design (just like with the radius, for example).